CLAUDE.md

```diff
@@ -117,8 +117,12 @@ For database migration and type generation workflows, see [`docs/migrationOpsCon
 - Generate CSS type definitions with `pnpm gen:css`
 - Watch mode available with `pnpm dev:css`
 
+## Pull Requests
+When creating pull requests, refer to @.github/pull_request_template.md for the required information and format.
```

Comment:
:eye: `@.github/pull_request_template.md` seems to be correct.  https://github.com/search?q=+%22%40.github%2Fpull_request_template.md%22&type=code